--- a/binder/binder.c
+++ b/binder/binder.c
@@ -2452,7 +2452,7 @@ static int binder_translate_binder(struc
 		ret = -EINVAL;
 		goto done;
 	}
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,2))
 	if (security_binder_transfer_binder(proc->cred, target_proc->cred)) {
 #else
 	if (security_binder_transfer_binder(proc->tsk, target_proc->tsk)) {
@@ -2502,7 +2502,7 @@ static int binder_translate_handle(struc
 				  proc->pid, thread->pid, fp->handle);
 		return -EINVAL;
 	}
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,2))
 	if (security_binder_transfer_binder(proc->cred, target_proc->cred)) {
 #else
 	if (security_binder_transfer_binder(proc->tsk, target_proc->tsk)) {
@@ -2594,7 +2594,7 @@ static int binder_translate_fd(u32 fd, b
 		ret = -EBADF;
 		goto err_fget;
 	}
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,2))
 	ret = security_binder_transfer_file(proc->cred, target_proc->cred, file);
 #else
 	ret = security_binder_transfer_file(proc->tsk, target_proc->tsk, file);
@@ -2996,7 +2996,7 @@ static void binder_transaction(struct bi
 			return_error_line = __LINE__;
 			goto err_invalid_target_handle;
 		}
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,2))
 		if (security_binder_transaction(proc->cred,
 						target_proc->cred) < 0) {
 #else
@@ -4952,7 +4952,7 @@ static int binder_ioctl_set_ctx_mgr(stru
 		ret = -EBUSY;
 		goto out;
 	}
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,2))
 	ret = security_binder_set_context_mgr(proc->cred);
 #else
 	ret = security_binder_set_context_mgr(proc->tsk);
